packages/lib/src/interactions/models.ts

```diff
@@ -52,3 +45,9 @@ export interface UseDragState {
   isDragging: boolean;
   startDrag: (evt: PointerEvent) => void;
 }
+
+export type MouseEventName = {
+  [Key in keyof GlobalEventHandlersEventMap]: GlobalEventHandlersEventMap[Key] extends MouseEvent
+    ? Key
+    : never;
+}[keyof GlobalEventHandlersEventMap];
```

Comment:
I did not come up with this... https://medium.com/dailyjs/typescript-create-a-condition-based-subset-types-9d902cea5b8c

Basically you go from `{ pointerdown: PointerEvent, resize: ResizeEvent, ... }`, to `{ pointerdown: PointerEvent, resize: never, ... }`, to `'pointerdown' | ...`.